server/protocols/wkd/src/main/java/org/apache/james/wkd/store/WebKeyDirectoryStore.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.james.wkd.store;
+
+public interface WebKeyDirectoryStore {
```

Comment:
This should likely be in `server/data/data-wkd`

Implementation of this class might need to be extracted in other maven modules like `server/data/data-wkd-memory`